File: Dot examples/ASP Book Examples/Book Examples/Chapter25/VB/App_Code/Movie.vb
Imports System
Imports System.Data
Imports System.Data.SqlClient
Imports System.Web.Configuration

Public Class Movie
    Public Shared Function GetMovies() As DataTable
        Dim conString As String = WebConfigurationManager.ConnectionStrings("Movies").ConnectionString
        Dim dad As New SqlDataAdapter("SELECT Title,Director FROM Movies", conString)
        Dim movies As New DataTable()
        dad.Fill(movies)
        Return movies
    End Function
End Class

File: Dot examples/ASP Book Examples/Book Examples/Chapter25/CS/App_Code/Movie.cs
using System;
using System.Data;
using System.Data.SqlClient;
using System.Web.Configuration;

public class Movie
{
    public static DataTable GetMovies()
    {
        string conString = WebConfigurationManager.ConnectionStrings["Movies"].ConnectionString;
        SqlDataAdapter dad = new SqlDataAdapter("SELECT Title,Director FROM Movies", conString);
        DataTable movies = new DataTable();
        dad.Fill(movies);
        return movies;
    }
}
